vault/login_mfa.go

```diff
@@ -83,7 +83,18 @@ func (b *SystemBackend) loginMFAPaths() []*framework.Path {
 			},
 			Operations: map[logical.Operation]framework.OperationHandler{
 				logical.UpdateOperation: &framework.PathOperation{
-					Callback:                  b.Core.loginMFABackend.handleMFALoginValidate,
+					Callback: b.Core.loginMFABackend.handleMFALoginValidate,
+					Responses: map[int][]framework.Response{
+						http.StatusOK: {{
+							Description: "OK",
+							Fields: map[string]*framework.FieldSchema{
+								"error": {
+									Type:     framework.TypeString,
+									Required: false,
+								},
```

Comment:
seems like `error` would only show up on non-200 responses. otherwise an empty response

Reply:
same point as above.

Reply:
nice catch, might need to update my PR's